### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by Iqbal-Elham**
## Hello microverse project
This is the first project of microverse
- I have used the Html and css 
- I used linters for beautiful coding and free of errors
- Finally I pushed in my github repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/iqbal-elham)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/iqbal-elham)
+
+
```
Comment (levy002) [inline README.md]: - [ ] Kindly make sure all main headers from the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) are presented in your README file. :wink:  I highly recommend to customize it according to your project.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Iqbal-Elham   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good job for using the correct github-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+This is the first project of Microverse
+
+## Built With
+
+- Html
+- Css
+
+
+
+
```
Comment (anagudelogu) [inline README.md]: - [ ] Great job documenting your code! 👍🏼 However, as stated by the last code reviewer, in order to provide professional and descriptive documentation, you could add some more information to your `README` file 🗒️  such as a `Getting started` section 🔧 , describing the steps that other people need to follow in order to get you code set and running. A `Contributing` section, describing how others can contribute to your project (usually, using issues). A `Support` section motivating people to support your project, and so on. 💯 

You're almost done! Please work on these minor changes and also take them in mind for your future projects! 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
Hi @Iqbal-Elham , 

Great work on making the changes requested by a previous reviewer 👏 🍾 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.



 ## Highlights
✅ Good Linters setup. 💯 
✅ Great GitHub flow. 👍🏼 
✅ Excellent pull request summary. 🥇 
✅ Descriptive commit messages. 👍🏼 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Iqbal-Elham , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**